### ISSUE TITLE
"Is this spot suitable to cross the road and is it legal to cross here?" may need to be reworded for the UK, at least.

### ISSUE THREAD

**Issue by TurnrDev** [streetcomplete/StreetComplete#3205]
![Screenshot_20210822-213656_Paint The Town Red Dev.png](https://user-images.githubusercontent.com/11667059/130369377-0e81c449-0ed0-4afa-9fc9-02a3a5a4b532.png)

Jaywalking is legal in the UK, so all spots are legal in the UK to cross. However, I would not want to add a crossing node unless there is at least some infrastructure to facilitate the crossing: dropped kurb etc

I think maybe the question should clarify there should be at least *some* form of infrastructure in place

**Comment by westnordost** [comment]
There needn't be any infrastructure there for it to be a crossing.

**Comment by 10992-osm** [comment]
If it is always legal, what sort of things would one be looking for in order to decide that something is not a crossing?

Presumably on a dual carriageway, if there is a barrier in the central reservation would be a no.

**Comment by TurnrDev** [comment]
> There needn't be any infrastructure there for it to be a crossing.

I'm just concerned that this quest is going to spam the UK with crossing nodes where there's no crossings

**Comment by westnordost** [comment]
With no crossings, you mean places where no crossing pedestrians would be expected? Because the screenshot you post, I'd say that drivers could expect people to cross the road at exactly the point where they come out of this path from behind the terraced houses.

But anyway, what to look for:
F.e. at the point where the footway meets the road, there is a lane (or parking bays) of closely parked cars, or there is some form of barrier

**Comment by westnordost** [comment]
The basic logic for when the quest is asked is to look for where a footway crosses a road.

**Comment by peternewman** [comment]
But if there is a pavement on both sides of the road, and a "proper" pedestrian crossing 50m up the road, that's where you *should* cross, but you *can* cross at that point.

For example a mother and baby/toddler would go to the crossing, but an impatient/lazy person will probably cross at that point.

Or someone going from 2 to 3 on Norman Road will probably walk directly across, but it doesn't mean there's a crossing there.

Won't the router use these crossings to tell people to cross there? In this particular example I wouldn't recommend it, due to the corner probably making it harder to see cars coming, but that doesn't mean I wouldn't do it myself.

**Comment by TurnrDev** [comment]
Yes, that's my point. In the UK, it's legal to cross at any position along the road. I also know that OSM uses crossing nodes as landmarks, in a way. You wouldn't want your GPS to say "Pedestrian crossing ahead" when there isn't. Even though the point is legal, and suitable for an able-bodied person with good senses, there is no crossing. At the very least, a crossing will have a lowered kerb.

**Comment by westnordost** [comment]
I get your point. 

Though, I have looked at in what instances `highway=crossing` and in particular `crossing=unmarked` is currently used a few weeks ago in various cities and found that this combination is used not at all only on "real" (i.e. where there is any infrastructure) crossings. See https://forum.openstreetmap.org/viewtopic.php?pid=835111#p835111

And anyway, you know how StreetComplete works: Something must be tagged to resolve it. `crossing=no` for your example, @TurnrDev , would also be wrong, because you **can** cross there.

Maybe don't ask for residential roads? On the other hand, I know enough residential roads that is crossed by a footway and there is a proper crossing there.

**Comment by TurnrDev** [comment]
I think the solution might be to just disable in the UK since Jaywalking isn't really a thing here?

Or I'm just moaning for nothing as I maintain my own personal fork anyway

**Comment by westnordost** [comment]
Jaywalking is not a thing in most countries except the US

**Comment by westnordost** [comment]
Do read the translation of the posted link, tgen you are up to speed and maybe have a different opinion.

**Comment by TurnrDev** [comment]
A solution to the problem I have with this could be done by asking if a curb is lowered at crossing or not. I think we already do that, but I think we lack an answer for "the curb is not lowered". "High Kerb" adds "kerb"=" raised" which would imply the kerb at the location is higher than at other places along the road (like you might find at a public transport stop)

"kerb"="regular" is probably the best tag for this, but it's not documented and only has 0.27% of uses. 🤔

**Comment by westnordost** [comment]
I'd say it would be `kerb=raised`. But I don't think it is correct to add information on the kerb to the intersection between a road and a footway, or is it?

**Comment by TurnrDev** [comment]
> I'd say it would be `kerb=raised`. 

Definitely not, the [wiki](https://wiki.openstreetmap.org/wiki/Key:kerb#Tagging) says that `kerb=raised` is "Raised above the norm". If the kerb to the left and right of the crossing are the same, it's just `kerb=regular`

> But I don't think it is correct to add information on the kerb to the intersection between a road and a footway, or is it?

True, it might cause routers to think theirs a barrier there. Can anybody confirm if any routers use the `kerb` tag when it's not paired with `barrier=kerb`?

A quick look on taginfo shows a 33% usage between `kerb=*` and `crossing=*` and a 2.9% usage from the crossing side

**Comment by matkoniecz** [comment]
> Definitely not, the wiki says that kerb=raised is "Raised above the norm". If the kerb to the left and right of the crossing are the same, it's just kerb=regular

1) that is `kerb=raised` - see further definition

2) `kerb=regular` was discussed and quite clearly rejected in https://wiki.openstreetmap.org/wiki/Proposed_features/kerb%3Dregular

2) I opened https://wiki.openstreetmap.org/wiki/Talk:Key:kerb#Clarify_.22Raised_above_the_norm.22 to make this description less misleading

**Comment by TurnrDev** [comment]
I like this solution! Sorry for making more work for you

**Comment by westnordost** [comment]
No, thank you for offering this suggestion!
